### ISSUE TITLE
Why the decoder frames in alignment is a fixed value? 

### ISSUE THREAD

**Issue by indexalice** [NVIDIA/tacotron2#382]
I am fine-tuning on my own dataset (English, duration ~3s) using a pre-trained model on LJSpeech.

It shows that it inherited some kind of alignment at the beginning. But I think it still tries to decode something after the text is finished. 

So I am confused about the blank green space in target Mel and predicted Mel. And why the "decoder frames" in the alignment is a fixed value?  Will it finally result in no alignment? Or should I do any other preprocessing on my dataset? 

Any response will be appreciated.

By the way, batch_size = 32

![屏幕截图 2020-06-26 16 22 23](https://user-images.githubusercontent.com/20719001/85832901-d9e30780-b7cb-11ea-86cf-1b69ae784a81.png)
![屏幕截图 2020-06-26 16 23 50](https://user-images.githubusercontent.com/20719001/85832911-dd768e80-b7cb-11ea-8352-53b9f6c5a2a2.png)

Here is the alignment learned on LJSpeech.
![屏幕截图 2020-06-26 16 46 49](https://user-images.githubusercontent.com/20719001/85833469-b8cee680-b7cc-11ea-83de-74ee2b995d4d.png)

**Comment by indexalice** [comment]
After 300 epochs, the alignment result is still the same. My own dataset for finetuning is about 3 hours (a little small). And I reduced the learning_rate to 2e5. Should I use a larger learning rate?

Any response will be appreciated.
![屏幕截图 2020-06-28 19 17 02](https://user-images.githubusercontent.com/20719001/85945952-09387680-b97c-11ea-87b2-3d03e0ee245a.png)
![屏幕截图 2020-06-28 19 17 59](https://user-images.githubusercontent.com/20719001/85945954-0ccbfd80-b97c-11ea-9418-381395f94306.png)
